meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/supervisor.conf

```diff
@@ -1,3 +1,4 @@
+# This file represents the last known version of the supervisor
```

Comment:
This is a little complicated because it is on the rootfs which is read only. Does it make sense this comment on a read only fs?